tests/cmdline/commands/test_status.py

```diff
@@ -84,3 +86,14 @@ def get_backend():
     assert profile.database_hostname in result.output
     assert str(profile.database_port) in result.output
     assert result.exit_code is ExitCode.CRITICAL
+
+
+@pytest.mark.usefixtures('aiida_profile')
+def tests_database_version(run_cli_command, manager):
+    """Test the ``verdi database version`` command."""
+    backend_manager = manager.get_backend_manager()
+    db_gen = backend_manager.get_schema_generation_database()
+    db_ver = backend_manager.get_schema_version_backend()
+
+    result = run_cli_command(cmd_status.verdi_status)
+    assert f'{db_gen} / {db_ver}' in result.output_lines[4]
```

Comment:
Why not just check `in result.output`? Now you make this test sensitive to added or removed lines, and that is not the point of the test right?

Reply:
Mmm no I think the old test was checking the ending of the lines so it was a leftover from that, I agree it is better to make it less sensitive to the rest of the output.

Reply:
(which means I changed it)